cluster-up/cluster/kind-k8s-1.17/README.md

```diff
@@ -0,0 +1,33 @@
+# K8S 1.17 in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.17 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart.
+The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=kind-k8s-1.17
+export KUBEVIRT_NUM_NODES=2 # master + one node
+make cluster-up
+```
+
+The cluster can be accessed as usual:
+
+```bash
+$ cluster-up/kubectl.sh get nodes
+NAME                        STATUS   ROLES    AGE    VERSION
+kind-1.17-control-plane   Ready    master   105s   v1.17.x
+kind-1.17-worker          Ready    <none>   71s    v1.17.x
```

Comment:
this line is misleading if you want to change it, since this bug is fixed
but no biggie

i just dont understand how it wont give u ipv6 , unless you have other kind config which doesnt config ipv6 ?
saw you reasked review, i will lgtm , feel free to change or not whatever you want , thanks